queries/filter_expression_with_length_property/selector

```diff
@@ -0,0 +1 @@
+$[?(@.length() == 4)]
```

Comment:
Did you mean to include the parentheses? (The button to add a suggested fix here has disappeared!)

Reply:
Thanks for catching that! Fixed.